app-i18n/tagainijisho/tagainijisho-1.2~20180610.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Open-source Japanese dictionary and kanji lookup tool"
+DESCRIPTION="Tagaini Jisho is a free, open-source Japanese dictionary and \
+kanji lookup tool that is available for Windows, MacOS X and Linux and aims \
+at becoming your Japanese study assistant. It allows you to quickly search \
+for entries and mark those that you wish to study, along with tags and \
+personal notes. It also let you train entries you are studying and follows \
```

Comment:
```diff
-personal notes. It also let you train entries you are studying and follows \
+personal notes. It also lets you train entries you are studying and follows \
```
I guess this typo comes from upstream :)
BTW, the description in https://github.com/Gnurou/tagainijisho also has the same typo:
```diff
-Finally, Tagaini let you produce printed material ...
+Finally, Tagaini lets you produce printed material ...
```